benchmark/cmake/modules/AddSwiftBenchmarkSuite.cmake

```diff
@@ -20,6 +20,58 @@ function(runcmd)
   set(${RUNCMD_VARIABLE} ${${RUNCMD_VARIABLE}} PARENT_SCOPE)
 endfunction(runcmd)
 
+function (swift_benchmark_library objfileout sibfileout)
+  cmake_parse_arguments(BENCHLIB "" "MODULE_PATH;SOURCE_DIR;OBJECT_DIR" "SOURCES;LIBRARY_FLAGS" ${ARGN})
+
+  precondition(BENCHLIB_MODULE_PATH)
+  precondition(BENCHLIB_SOURCE_DIR)
+  precondition(BENCHLIB_OBJECT_DIR)
+  precondition(BENCHLIB_SOURCES)
+
+  set(module_name_path "${BENCHLIB_MODULE_PATH}")
+  get_filename_component(module_name "${module_name_path}" NAME)
+  set(srcdir "${BENCHLIB_SOURCE_DIR}")
+  set(objdir "${BENCHLIB_OBJECT_DIR}")
+  set(sources "${BENCHLIB_SOURCES}")
+
+  set(objfile "${objdir}/${module_name}.o")
+  set(swiftmodule "${objdir}/${module_name}.swiftmodule")
+
+  precondition(objfileout)
+  add_custom_command(
+    OUTPUT "${objfile}"
+    DEPENDS ${stdlib_dependencies} ${sources}
+    COMMAND "${SWIFT_EXEC}"
```

Comment:
More of a stylistic nit-pick. When writing out multi-line commands I tend to indent another level on successive lines. IMO it's easier to visually parse, but just my opinion.

Reply:
Just to make sure I am understanding correctly, you mean indent the COMMAND below? (I.e. starting with benchlib library flags?)

Reply:
(i.e. I see what you mean. It makes it clear what are trailing arguments vs something for COMMAND itself).